Test/Framework/Audio/SoundEffectTest.cs

```diff
@@ -272,15 +272,12 @@ public void Ctor2()
         public void InstanceNotDisposedWhenGameDisposed()
         {
             var game = new Game();
-            var s = new SoundEffectInstance();
```

Comment:
You could create a dummy `SoundEffect` like I do here:

https://github.com/MonoGame/MonoGame/blob/develop/Test/Framework/Audio/SoundEffectTest.cs#L151

Then you can restore this part of the test.